patterns/ecs-v1/bro

```diff
@@ -1,13 +1,31 @@
 # https://www.bro.org/sphinx/script-reference/log-files.html
 
-# http.log
-BRO_HTTP %{NUMBER:ts}\t%{NOTSPACE:uid}\t%{IP:orig_h}\t%{INT:orig_p}\t%{IP:resp_h}\t%{INT:resp_p}\t%{INT:trans_depth}\t%{GREEDYDATA:method}\t%{GREEDYDATA:domain}\t%{GREEDYDATA:uri}\t%{GREEDYDATA:referrer}\t%{GREEDYDATA:user_agent}\t%{NUMBER:request_body_len}\t%{NUMBER:response_body_len}\t%{GREEDYDATA:status_code}\t%{GREEDYDATA:status_msg}\t%{GREEDYDATA:info_code}\t%{GREEDYDATA:info_msg}\t%{GREEDYDATA:filename}\t%{GREEDYDATA:bro_tags}\t%{GREEDYDATA:username}\t%{GREEDYDATA:password}\t%{GREEDYDATA:proxied}\t%{GREEDYDATA:orig_fuids}\t%{GREEDYDATA:orig_mime_types}\t%{GREEDYDATA:resp_fuids}\t%{GREEDYDATA:resp_mime_types}
+# TODO: move ZEEK_xxx under patterns/zeek ?!
+BRO_BOOL [TF]
+BRO_DATA [^\t]+
 
-# dns.log
-BRO_DNS %{NUMBER:ts}\t%{NOTSPACE:uid}\t%{IP:orig_h}\t%{INT:orig_p}\t%{IP:resp_h}\t%{INT:resp_p}\t%{WORD:proto}\t%{INT:trans_id}\t%{GREEDYDATA:query}\t%{GREEDYDATA:qclass}\t%{GREEDYDATA:qclass_name}\t%{GREEDYDATA:qtype}\t%{GREEDYDATA:qtype_name}\t%{GREEDYDATA:rcode}\t%{GREEDYDATA:rcode_name}\t%{GREEDYDATA:AA}\t%{GREEDYDATA:TC}\t%{GREEDYDATA:RD}\t%{GREEDYDATA:RA}\t%{GREEDYDATA:Z}\t%{GREEDYDATA:answers}\t%{GREEDYDATA:TTLs}\t%{GREEDYDATA:rejected}
+# http.log - old format (before the Zeek rename) :
+BRO_HTTP %{NUMBER:timestamp}\t%{NOTSPACE:[zeek][session_id]}\t%{IP:[source][ip]}\t%{INT:[source][port]:int}\t%{IP:[destination][ip]}\t%{INT:[destination][port]:int}\t%{INT:[zeek][http][trans_depth]:int}\t(?:-|%{WORD:[http][request][method]})\t(?:-|%{BRO_DATA:[url][domain]})\t(?:-|%{BRO_DATA:[url][original]})\t(?:-|%{BRO_DATA:[http][request][referrer]})\t(?:-|%{BRO_DATA:[user_agent][original]})\t(?:-|%{NUMBER:[http][request][body][bytes]:int})\t(?:-|%{NUMBER:[http][response][body][bytes]:int})\t(?:-|%{POSINT:[http][response][status_code]:int})\t(?:-|%{DATA:[zeek][http][status_msg]})\t(?:-|%{POSINT:[zeek][http][info_code]:int})\t(?:-|%{DATA:[zeek][http][info_msg]})\t(?:-|%{BRO_DATA:[zeek][http][filename]})\t(?:\(empty\)|%{BRO_DATA:[zeek][http][tags]})\t(?:-|%{BRO_DATA:[url][username]})\t(?:-|%{BRO_DATA:[url][password]})\t(?:-|%{BRO_DATA:[zeek][http][proxied]})\t(?:-|%{BRO_DATA:[zeek][http][orig_fuids]})\t(?:-|%{BRO_DATA:[zeek][http][orig_mime_types]})\t(?:-|%{BRO_DATA:[zeek][http][resp_fuids]})\t(?:-|%{BRO_DATA:[zeek][http][resp_mime_types]})
```

Comment:
Beats (for all Zeek modules) copies [`event.id`](https://github.com/elastic/beats/blob/v7.9.2/x-pack/filebeat/module/zeek/http/test/http-json.log-expected.json#L22) from [`zeek.session_id`](https://github.com/elastic/beats/blob/v7.9.2/x-pack/filebeat/module/zeek/http/test/http-json.log-expected.json#L67) ... wonder if we should skip `zeek.session_id` and only set `event.id` here (one confusing part would be that in [`BRO_FILES`](https://github.com/logstash-plugins/logstash-patterns-core/pull/286/files#diff-e6850acc553afe24076eb73346c41481b1bda28ce96f608b55fd2405f6d2a381R26) we have the related connection ids and match it as `zeek.files.session_ids`)

Reply:
Yeah I think it makes sense to keep `zeek.session_id` as a custom field, for the relationship with `zeek.files.session_ids`.

Users can always copy it if they want to map it further to ECS, and better align with Beats.

The other way around doesn't work, though, since `event.id` is meant to be one value, so we couldn't capture file's multiple session IDs with that approach.

So I agree with you on this one.

Reply:
Distinct comment on this same pattern. ECS 1.7 (to be released with stack 7.11) will include a `mime_type` field for both request and response. So I think you could migrate the mime_type fields here.

https://www.elastic.co/guide/en/ecs/1.7/ecs-http.html

Reply:
Thanks, was only looking at the 1.6 tag so missed that. Updated to use `http.request/response.mime_type`.